zebra/zserv.h

```diff
@@ -142,6 +142,8 @@ struct zserv {
 	uint32_t redist_v4_del_cnt;
 	uint32_t redist_v6_add_cnt;
 	uint32_t redist_v6_del_cnt;
+	uint32_t v4_route_nhg_add_cnt;
+	uint32_t v6_route_nhg_add_cnt;
```

Comment:
What about deletion/update counters?

Reply:
not implemented.
If I had to implement it, it would not be at ZEBRA level, as I don't have the info in ZAPI messages.